tools/deb/istio-iptables.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Initialization script responsible for setting up port forwarding for Istio sidecar.
+
+set -o nounset
+set -o pipefail
+IFS=,
+
+ISTIO_SIDECAR_CONFIG=${ISTIO_SIDECAR_CONFIG:-/var/lib/istio/envoy/sidecar.env}
+if [ ! -r $ISTIO_SIDECAR_CONFIG ]; then
+  echo "Missing sidecar config: $ISTIO_SIDECAR_CONFIG"
+  exit 1
+fi
+
+. $ISTIO_SIDECAR_CONFIG
+
+if [ -z "${ENVOY_UID:-}" ]; then
+  # Default to the UID of ENVOY_USER
+  ENVOY_UID=$(id -u ${ENVOY_USER:-istio})
+  if [ ! $? == 0 ]; then
+     echo "Invalid istio user $ENVOY_UID $ENVOY_USER"
+     exit 1
+  fi
+  ENVOY_UID=${ENVOY_UID},0
```

Comment:
Once again folks, this script is the crux of our entire system unfortunately. This is how we do the magical transparent proxying. we should have one shared across repos before final 0.2 release (can tolerate separate ones for couple of initial test releases). 
 However, I would like to keep the divergence between two versions to the absolute minimum possible. If variable names are not clear, let's fix in the pilot repo as well. If you want, I can take a crack at this script. 

cc @ayj @louiscryan @kyessenov
@tgraf FYI since you were asking about our pain points for transparent proxying.